modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -5133,6 +5137,55 @@ public void testUpdateUserSystemObjectWithJavaDelegateObjectValidationRule()
 		}
 	}
 
+	@FeatureFlags("LPD-32050")
+	@Test
+	public void testValidateRequiredValues() throws Exception {
```

Comment:
Brian [mentioned about the test method in his last review](https://github.com/brianchandotcom/liferay-portal/pull/160632#issuecomment-2779760484)

Reply:
Maybe the pattern of this class is > testAddObjectEntryWith....  (testAddObjectEntryWithLocalizedBooleanObjectField )

Reply:
@marcelabc he mentioned that because there was not `_validateRequiredValues` in Carol's pull request.

Reply:
I beleive it works for this change, what do you think?

Reply:
@MarinhoFeliphe I prefer to change the method name since it’s not clear that we are testing this method specifically (ValidateRequiredValues). In the test, we call ObjectEntryTestUtil.addObjectEntry(), so I think it would be better if the test name followed the same pattern as the others in this class.

If you need help renaming it, let me know! :)